inspirehep/modules/records/receivers.py

```diff
@@ -412,3 +414,21 @@ def populate_author_count(sender, json, *args, **kwargs):
         if 'supervisor' not in author.get('inspire_roles', [])
     ]
     json['author_count'] = len(authors_excluding_supervisors)
+
+
+def populate_authors_full_name_normalized(sender, json, *args, **kwargs):
+    """Populate the ``authors.full_name.raw`` field of Literature records."""
+    if 'hep.json' not in json.get('$schema'):
+        return
+
+    authors = json.get('authors', [])
+
+    for index, author in enumerate(authors):
+        full_name = author.get('full_name')
+        if full_name:
+            if not isinstance(full_name, unicode):
+                unicode_full_name = unicode(full_name, 'utf-8')
```

Comment:
Same goes for this, Python3 doesn't know the name `unicode`. 
Every `str`ing in Python 3 is converted into unicode code points.